Doc/library/logging.rst

```diff
@@ -1038,6 +1042,9 @@ functions.
    above example). In such circumstances, it is likely that specialized
    :class:`Formatter`\ s would be used with particular :class:`Handler`\ s.
 
+   The function will call :func:`basicConfig` if the root logger doesn't have
```

Comment:
Change to
```
This function (as well as :func:`info`, :func:`warning`, :func:`error` and :func:`critical`) will call ...
```

Reply:
Addressed on d173e5b75404a6dbf8883df24928f53795cef2ad